cloud-controller-manager/osc/tags_test.go

```diff
@@ -177,7 +178,7 @@ func TestHasClusterTag(t *testing.T) {
 		for k, v := range g.Tags {
 			ec2Tags = append(ec2Tags, &ec2.Tag{Key: aws.String(k), Value: aws.String(v)})
 		}
-		result := c.tagging.hasClusterTag(ec2Tags)
+		result := c.tagging.hasClusterAWSTag(ec2Tags)
```

Comment:
hasClusterOscTag ?

Reply:
Not yet migrated, it still uses EC2 tags